endoreg_db/serializers/patient_finding/patient_finding_classification.py

```
@@ -0,0 +1,21 @@
+from endoreg_db.serializers.misc.translatable_field_mix_in import TranslatableFieldMixin
+
+
+from rest_framework import serializers
+
+
+class PatientFindingClassificationSerializer(serializers.ModelSerializer, TranslatableFieldMixin):
+    """Serializer für PatientFinding-Klassifikationen"""
+
+    classification_name = serializers.SerializerMethodField()
+    classification_choice_name = serializers.SerializerMethodField()
+    class Meta:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)